public_html/layouts/resources/Fields.js

```diff
@@ -1478,18 +1479,20 @@ window.App.Fields = {
 		 */
 		toggleCheckBox(element) {
 			if ($(element).is(':checked')) {
```

Comment:
element is already jQuery object so it doesn't need to be wrapped by $()

Reply:
I don't remember why. Maybe some strange behavior in some browser when I cross-system / cross-browser tested... it's been a long time.
yes it's a jQuery obj